### ISSUE TITLE
Make script callable by other python scripts

### ISSUE THREAD

**Issue by micheleb** [vrde/slack-utils#2]
It'd be neat if I could call something like this from my python scripts:

``` python
import slack-post

slack-post.post_data(channel="random", icon-emoji=":+1:", text="bananas")
```

bonus points if the function name makes more sense (e.g. `slack-utils.post()` or even `slack.post()`) :+1:

**Comment by vrde** [comment]
Yup, nice idea! I'll create the `setup.py` this weekend and will improve the module to make it usable from other modules.
